projects/novo-elements/src/elements/date-picker/MultiDateInput.ts

```diff
@@ -127,16 +127,6 @@ export class NovoMultiDateInputElement implements OnInit, ControlValueAccessor {
 
   ngOnInit() {
     this.userDefinedFormat = this.format ? !this.format.match(/^(DD\/MM\/YYYY|MM\/DD\/YYYY)$/g) : false;
-    // if (!this.userDefinedFormat && this.textMaskEnabled && !this.allowInvalidDate) {
```

Comment:
Commented out code - if we leave this, we'll want to put a comment as to why it's commented and when we plan to uncomment.